src/app/faqs/faq-list/faq-list.component.html

```diff
@@ -1,142 +1,139 @@
-<div class="d-flex row-hl">
-  <!-- Category Selector -->
-  <div class="p-2 w-20">
-    <mat-card
-      class="mb-3 clickable"
-      matRipple
-      (click)="onClickTopCategory()"
-      [ngClass]="topOnly ? 'background-primary' : ''"
-    >
-      <mat-card-title>
-        <i class="fa fa-star"></i>
-        Top Questions</mat-card-title
-      >
-      <mat-card-subtitle
-        >The top questions we receive across all categories</mat-card-subtitle
-      >
-      <mat-card-content></mat-card-content>
-    </mat-card>
-    <hr />
-    <div *ngFor="let faqCategory of faqCategories">
+<div fxLayout="row" fxLayoutAlign="center center" class="k-header">
+  <button
+    mat-fab
+    color="primary"
+    matTooltip="Add Category"
+    (click)="onClickAddCategory()"
+  >
+    <mat-icon>add</mat-icon>
+  </button>
+  <div fxFlex class="k-header-title">FAQs</div>
+</div>
+
+<div class="mat-elevation-z8" class="p-4">
+  <div fxLayout="row">
+    <!-- Category Selector -->
+    <div fxFlex="20">
       <mat-card
         class="mb-3 clickable"
-        [ngClass]="
-          filterFaqCategory?.id == faqCategory.id ? 'background-primary' : ''
-        "
-        (click)="onClickCategory(faqCategory)"
         matRipple
+        (click)="onClickTopCategory()"
+        [ngClass]="topOnly ? 'background-primary' : ''"
       >
         <mat-card-title>
-          {{ faqCategory.title }}
-        </mat-card-title>
-        <mat-card-subtitle>
-          {{ faqCategory.description }}
-        </mat-card-subtitle>
+          <i class="fa fa-star"></i>
+          Top Questions</mat-card-title
+        >
+        <mat-card-subtitle
+          >The top questions we receive across all categories</mat-card-subtitle
+        >
         <mat-card-content></mat-card-content>
       </mat-card>
+      <hr />
+      <div *ngFor="let faqCategory of faqCategories">
+        <mat-card
+          class="mb-3 clickable"
+          [ngClass]="
+            filterFaqCategory?.id == faqCategory.id ? 'background-primary' : ''
+          "
+          (click)="onClickCategory(faqCategory)"
+          matRipple
+        >
+          <mat-card-title>
+            {{ faqCategory.title }}
+          </mat-card-title>
+          <mat-card-subtitle>
+            {{ faqCategory.description }}
+          </mat-card-subtitle>
+          <mat-card-content></mat-card-content>
+        </mat-card>
```

Comment:
I can't figure out why, but something that was changed is causing a bug that I had previously fixed here in the category cards. The matRipple is causing the padding inside the car to change when they are clicked. Might want to just create a bug ticket for this.